### ISSUE TITLE
Nethook2 Dumping null bytes 

### ISSUE THREAD

**Issue by c00kie17** [SteamRE/SteamKit#763]
![nethook](https://user-images.githubusercontent.com/10531093/67551242-1256b100-f726-11e9-88c5-95e5b80df7e4.PNG)
I have pulled the latest files and built them again.

**Comment by yaakov-h** [comment]
That means we don't know the name of the EMsg, but we do for those.

Latest source works for me:

![image](https://user-images.githubusercontent.com/426009/67618229-6e115f00-f838-11e9-83ce-d26ea5e60003.png)

**Comment by xPaw** [comment]
They removed emsg name from the struct in beta builds: https://github.com/m4dEngi/steamworks_dumper/commit/455c08db9859157231756d7e708110caa44b1e0e

emsg enum is in protos now: https://github.com/SteamDatabase/Protobufs/blob/master/steam/enums_clientserver.proto

**Comment by yaakov-h** [comment]
oh so it’s a beta thing. That would make sense.

**Comment by m4dEngi** [comment]
![2019-10-28 13_30_33-Friends List](https://user-images.githubusercontent.com/8540109/67672174-69f45700-f988-11e9-86f0-ae302405ee41.png)


> 
> 
> They removed emsg name from the struct in beta builds: [m4dEngi/steamworks_dumper@455c08d](https://github.com/m4dEngi/steamworks_dumper/commit/455c08db9859157231756d7e708110caa44b1e0e)
> 
> emsg enum is in protos now: https://github.com/SteamDatabase/Protobufs/blob/master/steam/enums_clientserver.proto

That's only valid for static analysis. It's still possible to retrieve string descriptors in runtime.

**Comment by c00kie17** [comment]
While running I get this message 
```
CCrypto::SymmetricEncryptChosenIV = 0x5cf9d470
Unable to find GetMessageList.
Detoured SymmetricEncryptChosenIV!
CWebSocketConnection::BBuildAndAsyncSendFrame = 0x5d01d900
CCMInterface::RecvPkt = 0x5ca3f320
Detoured CWebSocketConnection::BBuildAndAsyncSendFrame!
Detoured CCMInterface::RecvPkt!
```

**Comment by hudi1** [comment]
Any progress ? I have same issue that  c00kie17  when I built version of nethook from master branch

**Comment by yaakov-h** [comment]
See #765.

**Comment by yaakov-h** [comment]
Fixed by #765.
